Cue/app/tabs/library/play/PlayDeckSetupView.js

```diff
@@ -133,6 +210,7 @@ export default class PlayDeckSetupView extends React.Component {
         <ScrollView style={styles.container}>
```

Comment:
Is that just by doing `paddingBottom: 40`? Because `paddingBottom: '40pt'` does not work